src/GridList/GridTileTitlebar.js

```diff
@@ -0,0 +1,174 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiGridTileTitlebar', (theme) => {
+  const { mixins } = theme;
```

Comment:
I'm easy. I like the more succinct code when there are are lots of uses of the theme, but it's a small price to pay.

While where in that area of the code, what do you think regarding

```
      marginLeft: theme.mixins.gutters({}).paddingLeft,
      // TODO: Do we need left *and* right if they're always the same value?
      marginRight: theme.mixins.gutters({}).paddingRight,
```
Why not just a single `padding` value?